airflow/www/templates/airflow/dags.html

```diff
@@ -99,7 +99,8 @@ <h2>DAGs</h2>
                 <!-- Column 3: Name -->
                 <td>
                     <span>
-                      <a href="{{ url_for('Airflow.'+ dag.default_view, dag_id=dag.dag_id) }}" title="{{ dag.description[0:80] + '...' if dag.description|length > 80 else dag.description }}">
+                      <a href="{{ url_for('Airflow.'+ dag.default_view, dag_id=dag.dag_id) }}"
+                         title="{{ dag.description[0:80] + '...' if dag.description and dag.description|length > 80 else dag.description }}">
```

Comment:
Shouldn't we remove the title attribute if dag.description == None?

Reply:
Same as above, WDYT